### PR TITLE
feat!: remove Transifex calls for FC-0012 - OEP-58

### PR DESCRIPTION
**Pull request by shadinaif**
Breaking change!
---

This change breaks the Jenkins transifex integration which has been deprecated in favor of the new GitHub Transifex App integration as part of [OEP-58](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification).

Changes
---

- [x] Create a separate PR to remove source and language translations from the repositories, hence no `.po` or `.mo` files will be committed into the repos. https://github.com/openedx/xblock-free-text-response/pull/160
- [x] Removes direct use of `tx pull` and `tx push` commands from the [XBlock to let Open edX manage it](https://github.com/openedx/edx-platform/pull/33166).
- [x] Remove Transifex related `Makefile` targets and configuration files.
- [ ] ~~Use the OEP-58 JavaScript translations from the Open edX platform~~ NA
- [ ] ~~Remove `transifex-client` from requirements~~ NA

Merge timeline
---

This should only be merged after [Translation Infrastructure update OEP-58](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification) is fully implemented.

The timing announcement will be shared by @brian-smith-tcril on [#translations-project-fc-0012](https://openedx.slack.com/archives/C04R6TUJB7T) Open edX Slack channel.

**## Keep this pull request as a draft to prevent an accidental merge. ##**

Pre-merge checklist
---

- [ ] https://github.com/openedx/wg-translations/issues/20 is approved

References
---

This contribution is part of the [FC-0012 project](https://openedx.atlassian.net/l/cp/XGS0iCcQ) which is sparked by the [Translation Infrastructure update OEP-58](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification).

Up-to-date project overview and details are available in the [Approach Memo and Technical Discovery: Translations Infrastructure Implementation](https://docs.google.com/document/d/11dFBCnbdHiCEdZp3pZeHdeH8m7Glla-XbIin7cnIOzU/edit#) document.

Join the conversation on [Open edX Slack #translations-project-fc-0012](https://openedx.slack.com/archives/C04R6TUJB7T).

Check the links above for full information about the overall project.

### CODE REVIEW

**Comment by shadinaif** [comment]
@OmarIthawi , I kept `./freetextresponse/conf/locale/eo` directory because we have tests on dummy translations here

**Comment by shadinaif** [comment]
for your review @OmarIthawi

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+Unreleased changes are listed here.
```
Comment (OmarIthawi) [inline CHANGELOG.md:3]:
```suggestion
## Unreleased changes are listed here.
```

**Review by OmarIthawi — APPROVED** [review]
Thanks!! Looks good.

**Review by OmarIthawi — APPROVED** [review]

**Comment by OmarIthawi** [comment]
@shadinaif please set as `ready for merge` by removing `draft` status.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Unreleased changes are listed here.
```
Comment (brian-smith-tcril) [inline CHANGELOG.md:3]:
```suggestion
## Version 4.0.0
```

**Comment by OmarIthawi** [comment]
done @brian-smith-tcril

**Review by brian-smith-tcril — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@shadinaif 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/xblock-free-text-response&entry.867055334=https://github.com/openedx/xblock-free-text-response/pull/157&entry.1484655318=https://github.com/shadinaif&entry.752974735=2024-01-31+13:18&entry.1917517419=2024-03-18+18:55&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**